Dockerfile

```diff
@@ -3,6 +3,7 @@
 FROM quay.io/buildah/stable:v1.9.0
 COPY extract.sh .
 RUN yum -y install wget
```

Comment:
you could do this in one line `yum -y install wget git`

Reply:
Done :)